### ISSUE TITLE
Ensure that issues are displaying properly when filtered by label

### ISSUE THREAD

**Issue by MelSumner** [ember-learn/ember-help-wanted#112]
Currently, not all repos/labels are showing up when filtered. 

If you compare the issues that show up here: https://help-wanted.emberjs.com/learning?label=DecEmber you can see that according to the [issue filter code](https://github.com/ember-learn/ember-help-wanted-server/blob/master/src/issue-filter.js) more repos should be appearing in the list. 

(Note: this might be an issue for [ember-help-wanted-server](https://github.com/ember-learn/ember-help-wanted-server/).)

**Comment by prasannavijayan** [comment]
I am working on this issue.
Hi @MelSumner, 

I was checking this issue and found that learning category filters issues only these repository mentioned below,
```code
{ repo: 'ember-learn/ember-styleguide', labels: 'help wanted :sos:' },
{ repo: 'ember-learn/guides-source', labels: 'help wanted' }
```
I think we need to more repo which is in this category eg: `ember-api-docs`, `ember-website` and `ember-help-wanted`. 

By adding these repo the issues will be filtered accordingly and will display in respective issues route.

your thoughts?

**Comment by MelSumner** [comment]
sounds good! let's try it and see if it helps.

**Comment by MelSumner** [comment]
👍 fixed with PR #122
